plugin/cmd/kubectl/utils/utils.go

```diff
@@ -0,0 +1,72 @@
+package utils
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+)
+
+type namespace struct {
+	Metadata metadata `json:"metadata"`
+	Code     int      `json:"code"`
+}
+
+type metadata struct {
+	Name string `json:"name"`
+}
+
+func CheckNamespaceExists(name string) error {
+	proxyURL := "http://127.0.0.1"
+	proxyPort := "8881"
+
+	kubeProxy := exec.Command("kubectl", "proxy", "-p", proxyPort)
```

Comment:
Here you can use the `KUBECTL_PLUGINS_CALLER` env var, which will give you the full path to the `kubectl` binary.